### PR TITLE
connected to db

### PR DESCRIPTION
**Pull request by Edenlsw**
Thoughts
 This code connects to test database for rspec and main database for browser

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -17,6 +17,8 @@ class Kewtbnb < Sinatra::Base
     erb :spaces
   end
 
+
```
Comment (MakersKyrstenStone) [inline app.rb:20]: no actual change so ok with this

**Comment by tomhenebury** [comment]
This looks good to me :)

**Review by MakersKyrstenStone — APPROVED** [review]
seems fine to me

**Review by tomhenebury — COMMENTED** [review]
Looking good

**Pull request merged**